akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsSampleSpec.scala

```diff
@@ -45,7 +45,7 @@ object StatsSampleSpecConfig extends MultiNodeConfig {
           cluster {
             enabled = on
             allow-local-routees = on
-            use-role = compute
+            use-role-set = ["compute"]
```

Comment:
I first thought of using ```use-roles```, but I think it does not convey well that it is meant to be all roles that need to be matched, in other words a set of roles. That said, ```use-roles``` may be more idiomatic with Akka configuration nomenclature in general.

As for Java API, would varargs replace existing Java ```Set<String>``` stubs or are they in addition to those stubs ?

I agree that backward compatibility can be challenging without polluting the API. I could add back support for ```use-role```, mark it as deprecated in code and in documentation, only mention ```use-role-set``` ?